pages/docs/tutorials/getting-started/request_reply.md

```diff
@@ -0,0 +1,240 @@
+---
+title: Request and reply pattern
+weight: 20
+---
+
+A quite common messaging pattern is [request-reply](https://www.enterpriseintegrationpatterns.com/patterns/messaging/RequestReply.html), which describes a **requester**, which sends a request message and waits for a reply and a **replier** which receives the request and responds with a reply.
+
+As of AsyncPAI v3, this pattern can be described natively.
+ 
+# Describing a requester
+
+We are gonna use a very simple ping and pong example where a requestor sends the ping and responder respond with pong. Take notice this is an application-level ping and pong and is not related to ping/pong in standards such as WebSocket.
+
+To describe a **requester** in AsyncAPI we make use of an operation that `send`s the ping and expects a `reply` over `pong`.
+
+```yml
+asyncapi: 3.0.0
+
+info:
+  title: Ping/pong example with the requester
+  version: 1.0.0
+  description: Simple example with a requester that initiates the request-reply pattern.
+
+channels:
+  ping:
+    address: /
```

Comment:
Personal opinion:

I feel using different addresses for each channel will let people understand the concept better. `/ping` and `/pong` or any similar will, IMHO, let the user don't waste time trying to understand how this will work with both having `/` as address.

I know you  are describing other patterns [later on](https://github.com/asyncapi/website/pull/2071/files#diff-5ceb1af05da46640b00d0804db27ef434197db72733ab27b4c5255b2544cbe72R77), however it might make sense to use an example that is more focused on EDA rather than HTTP/Websocket, as it feels more aligned with what AsyncAPI target is. 
That means that maybe it makes sense to change the example to a real-world EDA scenario, like a business operation (Buy/Confirm) or similar.

Reply:
Yea that makes sense, let me change the initial example 🙂

Reply:
What would be the most common request/reply pattern that we can start off with?

Changing the first example to something else is quite a re-structuring of the entire flow and text, so I think making the flow of the text clear first before re-doing it makes sense.

Any suggestion?

Reply:
Take this just as a suggestion, I usually like examples based in common-life things like payments.

This is an example I usually use. Feel free to take it even though I would suggest others to share their opinion. Especially people that use that pattern in their systems like @GreenRover :

```yaml
asyncapi: '3.0.0'
info:
  title: Payment Service
  version: 1.0.0
  description: This service is in charge of processing payments
channels:
  ExecutePayment:
    address: payment/exec
    messages:
      ExecutePayment:
        $ref: '#/components/messages/ExecutePayment'
  PaymentProcessed:
    address: payment/done
    messages:
      PaymentProcessed:
        $ref: '#/components/messages/PaymentProcessed'
operations:
  ExecutePayment:
    action: receive
    channel:
      $ref: '#/channels/ExecutePayment'
    reply:
      channel:
        $ref: '#/channels/PaymentProcessed'
components:
  messages:
    ExecutePayment:
      payload:
        type: object
        properties:
          order:
            type: string
            description: Id of the order
          paymentData:
            type: string
            description: Encrypted payment data
    PaymentProcessed:
      payload:
        type: object
        properties:
          order:
            type: string
            description: Id of the order
          status:
            type: string
            enum: 
              - confirmed
              - errored
```

Reply:
Looks like a good example.
Missing only header. I will provide next week some examples.

Reply:
I changed the examples around to use the one with two different addresses first and the same channel later down the list.